tests/Sdk/Integration/Context/SpanContextTest.php

```diff
@@ -19,7 +19,7 @@ class SpanContextTest extends TestCase
     public function testInvalidSpan(string $traceID, string $spanID, string $errorRegex): void
     {
         $this->expectException(InvalidArgumentException::class);
-        $this->expectExceptionMessageRegExp($errorRegex);
+        $this->expectExceptionMessageMatches($errorRegex);
```

Comment:
When I left this as the previous value, all of our unit tests pass.  Was this intended to be changed to test PHPUnit, or something else?

For reference, the errors I was getting with `expectExceptionMessageMatches` are as follows:
https://gist.github.com/bobstrecansky/ca3eb2dac6eb37448ac1c9b73733e3a6

Reply:
expectExceptionMessageRegExp has been deprecated and removed from phpunit 9 so I replaced it with a newer one that is acualy the same. Github action fails and that is wierd, there may be some cache. I drafted this PR for now until I find out the reason

Reply:
after I ran a `make update`, my tests passed.  Perhaps it's a cached version of our build image?

Reply:
Rerunning the CI suite to validate.

Reply:
Aha!  The image that we are using uses the `shivammathur/setup-php` github action helper.  We can explicitly call out the version of phpunit that we want to use there; this is most likely the conflict.

https://github.com/shivammathur/setup-php#wrench-tools-support

We would probably just need to define that in our actions workflow:
https://github.com/open-telemetry/opentelemetry-php/blob/master/.github/workflows/php.yml#L27

Reply:
Is there any particular reason why composer.lock is gitignored?

Reply:
@nenad removed it here: https://github.com/open-telemetry/opentelemetry-php/pull/51